src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -24,394 +24,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import json
-from io import StringIO
-from pathlib import Path
-from typing import Any, List, Union
-
-import numpy as np
 import pytest
-from genai_perf.llm_metrics import LLMMetrics, LLMProfileDataParser, ResponseFormat
-from genai_perf.tokenizer import DEFAULT_TOKENIZER, get_tokenizer
-
-
-def ns_to_sec(ns: int) -> Union[int, float]:
-    """Convert from nanosecond to second."""
-    return ns / 1e9
-
-
-class TestLLMProfileDataParser:
```

Comment:
Where did this testing migrate to?

Reply:
Wow thanks for catching this! I forgot to add the migrated test file 😅